packages/components/src/Chips/InternalChip.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import classnames from "classnames";
+import styles from "./InternalChip.css";
+import { useAssert } from "./useAssert";
+import { InternalChipProps } from "./ChipTypes";
+import { InternalChipAffix } from "./InternalChipAffix";
+import { Typography } from "../Typography";
+import { Tooltip } from "../Tooltip";
+
+export function InternalChip({
+  label,
+  active = false,
+  disabled = false,
+  invalid = false,
+  prefix,
+  suffix,
+  warnOnLongLabels = false,
+  onClick,
+}: InternalChipProps) {
+  const [truncateRef, setTruncateRef] = useState<HTMLElement | null>();
+  const isClickable = onClick && !disabled;
+  const classNames = classnames(styles.chip, {
+    [styles.clickable]: isClickable,
+    [styles.active]: active,
+    [styles.disabled]: disabled,
+    [styles.invalid]: invalid,
+    [styles.hasPrefix]: prefix,
+    [styles.hasSuffix]: suffix,
+  });
+  const affixProps = {
+    active: active,
+    invalid: invalid,
+    disabled: disabled,
```

Comment:
with modern JS we can do this for the same result `const affixProps = { active, invalid, disabled };`

Where if the `key` and `value` of the object are the same, we don't need to specify.

Reply:
ah yea, forgot that that's a thing now